src/main/java/uk/gov/ida/eventemitter/ResiliencyFileWriter.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.ida.eventemitter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ResiliencyFileWriter {
+    private static final Logger LOG = LoggerFactory.getLogger(ResiliencyFileWriter.class);
+
+    private final ObjectMapper objectMapper;
+
+    @Inject
+    public ResiliencyFileWriter(final ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    public void writeEvent(Event event) {
+        try {
+            String serialisedEvent = objectMapper.writeValueAsString(event);
+            LOG.info(serialisedEvent);
```

Comment:
This class isn't really doing a lot - we could just do `LOG.info(objectMapper.writeValueAsString(event)` in the EventEmitter class couldn't we? Especially as we aren't really handling the exception

Reply:
This class and its unit test class were imported from EventSink in ida-hub-support. Since this class is not doing a lot, I have replaced these classes with just one line  `LOG.info(objectMapper.writeValueAsString(event));` in the fourth commit.